.test_runner_config.yaml

```diff
@@ -62,6 +62,7 @@ steps:
   - ipa-test-config --help
   - ipa-test-task --help
   - ipa-run-tests ${tests_ignore} -k-test_dns_soa ${tests_verbose} ${path}
+  - '! grep -n -B10 BytesWarning /var/log/httpd/error_log'
```

Comment:
There is one check that is in py2 (I don't remember from top of my head), but majority of checks are in py3